pyanaconda/modules/boss/installation.py

```diff
@@ -0,0 +1,157 @@
+        else:
+            self._create_logs_directory()
+            self._copy_tmp_logs()
+            self._copy_pre_script_logs()
+            self._copy_dnf_debugdata()
+            self._copy_post_script_logs()
+            self._dump_journal()
+            self._chmod_logs()
+
+            log.info("Done.")
+
+    def _create_logs_directory(self):
+        """Create directory for Anaconda logs on the install target"""
+        util.mkdirChain(self._sysroot + "/var/log/anaconda/")
+
+    def _copy_tmp_logs(self):
+        """Copy a number of log files from tmp"""
+        LOG_FILES_TO_COPY = [
+            "anaconda.log", "syslog", "X.log", "program.log", "packaging.log", "storage.log",
+            "ifcfg.log", "lvm.log", "dnf.librepo.log", "hawkey.log", "dbus.log"
+        ]
+        for logfile in LOG_FILES_TO_COPY:
+            self._copy_file("/tmp/" + logfile,
+                            ANACONDA_LOG_DIR + logfile)
+        # copy also list of packages used for creating the installation media
+        self._copy_file("/root/lorax-packages.log",
+                        ANACONDA_LOG_DIR + "lorax-packages.log")
+
+    def _copy_pre_script_logs(self):
+        """Copy logs from %pre scripts"""
+        self._copy_tree("/tmp/pre-anaconda-logs",
+                        ANACONDA_LOG_DIR)
+
+    def _copy_dnf_debugdata(self):
+        """Copy DNF debug data"""
+        self._copy_tree("/root/debugdata",
+                        ANACONDA_LOG_DIR + "dnf_debugdata/")
+
+    def _copy_post_script_logs(self):
+        """Copy logs from %post scripts"""
+        for logfile in glob.glob("/tmp/ks-script*.log"):
+            self._copy_file(logfile,
+                            ANACONDA_LOG_DIR + os.path.basename(logfile))
+
+    def _dump_journal(self):
+        """Dump journal from the installation environment"""
+        with open(self._sysroot + ANACONDA_LOG_DIR + "journal.log", "w") as logfile:
+            util.execWithRedirect("journalctl", ["-b"], stdout=logfile)
+
+    def _chmod_logs(self):
+        """Set access bits"""
+        util.execInSysroot("bash", ["-c", "chmod 0600 " + ANACONDA_LOG_DIR + "*"])
+
+    def _copy_kickstart(self):
+        """Copy generated kickstart file"""
+        log.info("Copying generated kickstart file...")
+
+        NOSAVE_INPUT_KS_FILE = "/tmp/NOSAVE_INPUT_KS"
+
+        if os.path.exists(NOSAVE_INPUT_KS_FILE):
+            log.info("Nosave used, skipping.")
+            os.remove(NOSAVE_INPUT_KS_FILE)
+        else:
+            self._copy_file("/run/install/ks.cfg",
+                            "/root/original-ks.cfg")
+            log.info("Done.")
+
+    def _relabel_files(self):
+        """Relabel the anaconda logs.
+
+        The files we've just copied could be incorrectly labeled, like hawkey.log:
+        https://bugzilla.redhat.com/show_bug.cgi?id=1885772.
+        """
+        util.execInSysroot("restorecon", ["-ir", ANACONDA_LOG_DIR])
+
+    def _copy_file(self, src, dest):
```

Comment:
I would rename this to `_copy_file_to_sysroot()` or something similar to be obvious that the paths pointed as dest argument are in the sysroot.

Reply:
Doing the same with the other function too.